src/adapter/mysql.cr

```diff
@@ -20,8 +20,8 @@ class Granite::Adapter::Mysql < Granite::Adapter::Base
   def select(table_name, fields, clause = "", params = [] of DB::Any, &block)
     statement = String.build do |stmt|
       stmt << "SELECT "
-      stmt << fields.map { |name| "#{table_name}.#{name}" }.join(", ")
-      stmt << " FROM #{table_name} #{clause}"
+      stmt << fields.map { |name| "#{quote_table_name(table_name)}.#{quote_column_name(name)}" }.join(", ")
```

Comment:
It might be possible to shorten this syntax a little by providing something like this in the `Adapter::Base`:

```crystal
   def col(name : String)
      quote_column_name name
   end
```

That would make all of these statements from `quote_column_name(...)` into just `col(...)`.

Reply:
Yep. I love shorter method name! 😺 

But I'm afraid that we can't image **quoting feature** from the name of `col`. Well, how about using `quote`? In fact, both quoting logic about columns and tables are same for all three adapters. And I felt it redundant. So we can merge them into one `quote` method.